File: Easy/Java/1295. Find Numbers with Even Number of Digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        
        for(int i : nums){
            int digits = 0;
            while(0 < i){
                digits++;
                i /= 10;
            }
            if(digits % 2 == 0)
                count++;
        }

        return count;
    }
}

File: Easy/Python/1295. Find Numbers with Even Number of Digits.py
class Solution(object):
    def findNumbers(self, nums):
        count = 0

        for i in nums:
            digits = 0
            while(0 < i):
                digits += 1
                i = i / 10
            if digits % 2 == 0:
                count += 1

        return count
